src/sql/src/plan/query.rs

```diff
@@ -1847,32 +1846,109 @@ pub fn plan_expr<'a>(ecx: &'a ExprContext, e: &Expr) -> Result<CoercibleScalarEx
     })
 }
 
-/// Plans a list of expressions such that all input expressions will be cast to
-/// the same type. If successful, returns a new list of expressions in the same
-/// order as the input, where each expression has the appropriate casts to make
-/// them all of a uniform type.
+/// Plans a list of expressions.
```

Comment:
Using "list" here makes it sound like it might only apply to the `list` type; maybe amend to "slice" or "set" or something even clearer?

Reply:
Probably not worth going back through CI for this!

Reply:
No, happy to! Thanks for the quick review.